cpp/include/proxy_sequences.h

```diff
@@ -174,6 +197,13 @@ class Sequence {
     return *this;
   }
 
+  Sequence &add_upload(const std::string &name,
+                       const std::shared_ptr<std::vector<char>> blob,
+                       const alba::Checksum *cs_o) {
+    _updates.push_back(
+        std::shared_ptr<Update>(new UpdateUploadObject(name, blob, cs_o)));
```

Comment:
using `std::make_shared<UpdateUploadObject>(name, blob, cs_o)` is preferrable (also applies to the existing calls):
https://github.com/isocpp/CppCoreGuidelines/blob/master/CppCoreGuidelines.md#r22-use-make_shared-to-make-shared_ptrs